test/mercury/resource-validation.test.ts

```diff
@@ -66,10 +63,35 @@ describe("resource checking tests", () => {
     const doc = getHappySpec();
     renameKey(doc, "/resource", "/this_resource");
     const result = await validateFile(renderSpecAsFile(doc), PROFILE);
-    breaksOnlyOneRule(
-      result,
-      "http://a.ml/vocabularies/data#resource-name-validation"
-    );
+    breaksTheseRules(result, [NAME_VALIDATION, NAME_VALIDATION]);
+  });
+
+  it("conforms when resource contains multiple literal parts", async () => {
+    const doc = getHappySpec();
+    renameKey(doc, "/resource", "/path/is/ok");
+    const result = await validateFile(renderSpecAsFile(doc), PROFILE);
+    conforms(result);
+  });
+
+  it("conforms when resource contains multiple literal parts and template", async () => {
```

Comment:
Should there also be a test for multiple templates?

Reply:
done